ios_widget_catalog_compare/.gitignore

```diff
@@ -0,0 +1,41 @@
+# Miscellaneous
```

Comment:
Strange that the .DS_Store file is still there. Maybe git rm --cached .DS_Store manually